core/controllers/skill_editor.py

```diff
@@ -146,6 +146,70 @@ class EditableSkillDataHandler(base.BaseHandler):
 
     GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
 
+    URL_PATH_ARGS_SCHEMAS = {
+        'skill_id': {
+            'schema': {
+                'type': 'basestring',
+                'validators': [{
+                    'id': 'is_regex_matched',
+                    'regex_pattern': constants.ENTITY_ID_REGEX
+                }]
+            }
+        }
+    }
+
+    HANDLER_ARGS_SCHEMAS = {
+        'GET': {},
+        'PUT': {
+            'version': {
+                'schema': {
+                    'type': 'int',
+                }
```

Comment:
Here I did not put the validator for "version" because one test case (test_editable_skill_handler_put_fails) was expecting status 404 when cannot get skill by id and in that they are passing version as -1. If I validate the version then it is going to return 400.

Reply:
Put it here and change the tests.

Reply:
Done.